warehouse/manage/views.py

```diff
@@ -729,13 +731,25 @@ def delete_project_role(project, request):
     permission="manage:project",
 )
 def manage_project_history(project, request):
-    journals = (
+    try:
+        page_num = int(request.params.get("page", 1))
+    except ValueError:
+        raise HTTPBadRequest("'page' must be an integer.") from None
```

Comment:
I believe you don't need `from None` here

Reply:
I removed `from None` here.